packages/ir-sdk/fern/apis/ir-types-latest/definition/http.yml

```diff
@@ -225,6 +225,12 @@ types:
 
   # examples
 
+  HttpEndpointExample:
+    discriminant: exampleType
+    union:
+      provided: ExampleEndpointCall
```

Comment:
nit: some docs here might be helpful or renaming the union key to `userProvided`

Reply:
Updated!